cmake/cvars.cmake

```diff
@@ -0,0 +1,50 @@
+set(CVAR_VSYNC_ENABLED "gVsyncEnabled" CACHE STRING "")
+set(CVAR_Z_FIGHTING_MODE "gZFightingMode" CACHE STRING "")
+set(CVAR_NEW_FILE_DROPPED "gNewFileDropped" CACHE STRING "")
+set(CVAR_DROPPED_FILE "gDroppedFile" CACHE STRING "")
```

Comment:
not part of this PR, just a note that we should make an issue to not need to use cvars for this

Reply:
https://github.com/Kenix3/libultraship/issues/535